Description: Use local js files
 Using Debian tools is fine, except when some files are missing (See #755468). 
 .
 When the above bug is fixed, just remove this patch.
Author: dod
Applied-Upstream: NA
--- /dev/null
+++ b/amd-externs.js
@@ -0,0 +1,8 @@
+/**
+ * @param {string} id
+ * @param {Array.<string>} dependencies
+ * @param {Function} factory
+ */
+function define(id, dependencies, factory) {}
+/** @type {*} */
+define.amd;
--- /dev/null
+++ b/console-externs.js
@@ -0,0 +1,5 @@
+var console = {};
+/**
+ * @param {string} message
+ */
+console.warn = function (message, var_args) {};
--- a/Makefile
+++ b/Makefile
@@ -43,12 +43,16 @@
 	      | grep -v total; \
 	done
 	@$(CLOSURE_COMPILER) --js src/prettify.js \
+		--externs console-externs.js \
+		--externs amd-externs.js \
 	    --define IN_GLOBAL_SCOPE=true \
 	    --output_wrapper='!function(){%output%}()' \
 	    > $(TAR_ROOT)/prettify.js
 	@wc -c src/prettify.js $(TAR_ROOT)/prettify.js \
 	    | grep -v total
 	@$(CLOSURE_COMPILER) --js src/run_prettify.js \
+		--externs console-externs.js \
+		--externs amd-externs.js \
 	    --define IN_GLOBAL_SCOPE=false \
 	    --output_wrapper='!function(){%output%}()' \
 	    > $(TAR_ROOT)/run_prettify.js
